tgui/packages/tgui/interfaces/PreferencesMenu/preferences/species/hemophage.ts

```diff
@@ -0,0 +1,51 @@
+import { Species } from "./base";
+
+const Hemophage: Species = {
+  description: "A crewmember afflicted with Hemophagia. Whether it's supernatural or purely through science, is still unknown.",
+  features: {
+    good: [{
+      icon: "bed",
+      name: "Locker Brooding",
+      description: "Hemophages can delay The Thirst and heal by resting in a \
+        rectangular-shaped object. So THAT'S why they do that!",
```

Comment:
This should be better.
```suggestion
      description: "Hemophages can delay their Thirst and mend their injuries by \
        resting in a rectangular-shaped object. So THAT'S why they do that!",
```